frontend/src/components/ProjectCard/projectCardStyle.ts

```diff
@@ -67,6 +67,20 @@ export const TypeBadge = styled('div')`
 
 export const CardDetail = styled('div')`
   margin-block-start: 1em;
+  list-style-type: none;
+
+  .kebab-menu-options {
```

Comment:
Suggest using goober instead of html classes.

```ts
export const KebabMenuOptions = styled('div')`
  style options
`
```

Then you can use:
```tsx
<KebabMenuOptions></KebabMenuOptions>
```
in the react code.